tensorrt_llm/_torch/disaggregation/native/bounce/buffer.py

```
@@ -0,0 +1,157 @@
+_MIB = 1024 * 1024
+
+
+def _div_up(a: int, b: int) -> int:
+    return (a + b - 1) // b
+
+
+class Buffer:
+    """One contiguous fabric-VMM region for coalescing KV; phys_chunk_size MUST
+    equal the KV pool's chunk size for the C++ VmmDescSplitter uniform-chunk invariant."""
+
+    __slots__ = ("_vm", "_size", "_device_id", "_name")
```

Ruff (0.15.18)

[warning] `Buffer.__slots__` is not sorted

Apply a natural sort to `Buffer.__slots__`

(RUF023)